component-samples/demo/owner/service.yml

```diff
@@ -43,6 +43,9 @@ owner:
        path: replacement.p12
        store-type: PKCS12
        password: $(encrypt_password)
+   to0-scheduler:
+     thread-count: 5
+     interval: 120
```

Comment:
Multiple RVInfo is not working with this PR.

Reply:
fixed in commit 912a8c7